components/Branding/Logo/Variants.js

```diff
@@ -0,0 +1,60 @@
+import * as colors from 'common/styles/themeMap';
+/* eslint-disable class-methods-use-this */
+export class Variants {
```

Comment:
Can we just do a function like `getVariantStyles` which is a switch case that returns the object needed? Trying to avoid `class` entirely in this codebase for newer devs.

Very sleek code though!

Reply:
We can I suppose but its quite a bit of squeeze for not much more juice. What about having this be the useful implementation of a class instead of avoiding a part of the language?

Reply:
I think avoiding parts of a language is totally fine. I would say it's a squeeze for _no_ juice. The only benefit is for newer readers of code, not for performance or anything like that.